### ISSUE TITLE
Windows Home edition

### ISSUE THREAD

**Issue by holtjonathan** [paytonrules/learn-reactive-extensions#1]
Hello,

I am unfortunately stuck with my wife's laptop for the week of Codemash.  She only has Win10 Home edition and Docker requires Pro.  So I will unable to do any of the Docker related tasks.

What can I do to still be able to follow along and benefit from this class?  I am very much interested in attending.

**Comment by paytonrules** [comment]
If you can install node and yarn, you can run the app locally.

Unfortunately I don't know how to set those up, as I'm a mac user myself. I only setup Docker to avoid platform-related issues.

**Comment by bep713** [comment]
Hey, I'm running on win 10 home too. 

I was able to install docker toolbox which instead of using Hyper V it relies on a Virtual box VM...but seems to get the job done. 

I had to work with it a bit to get it to work...but the biggest issue was that I needed to restart my PC after installing docker toolbox or it gave me an error when running any command.

The only thing that I can't get working atm is the app auto-refreshing when I save the puzzles.ts file, not sure if that's a docker toolbox issue or not.

Docker toolbox: https://docs.docker.com/toolbox/toolbox_install_windows/
